test/nuts/specialTypes/translation.nut.ts

```diff
@@ -161,24 +162,76 @@ describe('translations', () => {
       });
 
       describe('individual type retrieves', () => {
-        it('can retrieve COT', async () => {
+        it('can retrieve COT from CFT', async () => {
           execCmd(`project retrieve start -d ${translationPath} --json`, {
             ensureExitCode: 0,
           });
         });
 
         it('can retrieve COT from directory', async () => {
-          execCmd(
-            `project retrieve start -d ${path.join(
-              translationPath,
-              'customObject__c-es.objectTranslation-meta.xml'
-            )} --json`,
-            {
-              ensureExitCode: 0,
-            }
+          // the .objectTranslation is in my-app
+          const objectTranslationPath = path.join(
+            session.project.dir,
+            'my-app',
+            'main',
+            'default',
+            'objectTranslations',
+            'customObject__c-es',
+            'customObject__c-es.objectTranslation-meta.xml'
           );
+
+          execCmd(`project retrieve start -d ${objectTranslationPath} --json`, {
+            ensureExitCode: 0,
+          });
```

Comment:
This is retrieving a COT by filename, not directory.  Also, I think we can't rely on just the exit code 0 as a success.  I think we need to ensure the retrieve was written to the correct place and not the default location.